docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -111,60 +118,53 @@ on the ages and sexes to which the cause applies.
 Vivarium Modeling Strategy
 --------------------------
 
-.. todo::
+Scope
++++++
+
+This Vivarium modeling strategy is intended to simulate TBL cancer incidence/morbidity as well as mortality so that it reflects the estimates and assumptions of GBD.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+This model will assume the existence of a "recovered" cause model state in an attempt to be consistent with the GBD assumption that no morbidity due to TBL cancer does not occur more than ten years past incidence. Notably, since GBD performs the TBL cancer fatal and non-fatal models separately, this assumption in GBD applies to the non-fatal model only and not the fatal model, which is a limitation of our strategy. The assumption also asserts that there is no recurrance of TBL cancer.
```

Comment:
"no morbidity does not occur" - double negative that I think means something different than you intended